### PR TITLE
Providing full response

### PR DESCRIPTION
**Pull request by v-tec2706**
**Added:**
 -> mechanism for providing full known response 
 -> new methods in statement utils

### CODE REVIEW

File: src/common_utils/language_utils/sentence_filter_utils.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import src.common_utils.constants as constants
-from src.common_utils.database_service import DatabaseProxy
+import configuration as configuration
```
Comment (Andrzej97) [inline src/common_utils/language_utils/sentence_filter_utils.py]: Has same effect as just 'import configuration'?

File: src/connectivity/rest/rest_controller.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from flask import Flask
+from flask_restful import Resource, Api
+from flask_jsonpify import jsonify
+
+from src.common_utils.database.database_service import DatabaseProxy
+import configuration as configuration
```
Comment (Andrzej97) [inline src/connectivity/rest/rest_controller.py]: Has same effect as just 'import configuration'?

File: src/connectivity/robot/pepper_speech_recognition.py [hunk 3]
```diff
@@ -11,8 +10,8 @@
 from six.moves import queue
 
 import numpy as np
-from src.connectivity.socket_conn_robot import DataExchangeModule
-import src.common_utils.constants as constants
+from src.connectivity.socket.socket_conn_robot import DataExchangeModule
+import configuration as configuration
```
Comment (Andrzej97) [inline src/connectivity/robot/pepper_speech_recognition.py]: Has same effect as just 'import configuration'?

File: src/main_chat/chatbot_manager.py [hunk 4]
```diff
@@ -1,18 +1,22 @@
+import configuration as configuration
```
Comment (Andrzej97) [inline src/main_chat/chatbot_manager.py]: Has same effect as just 'import configuration'?

Comment (v-tec2706) [reply src/main_chat/chatbot_manager.py]: true

File: src/main_chat/response_continuation.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import configuration
+import src.common_utils.language_utils.statement_utils as statement
+from src.common_utils.database.database_service import DatabaseProxy
+from src.common_utils.language_utils.sentence_filter_utils import SentenceFilter
+
+
+def initialize_db_with_continue_statements():
+    # fixme: to be removed after adding to database initialization file
+
+    db = DatabaseProxy('mongodb://localhost:27017/', 'PepperChatDB')
```
Comment (rpych) [inline src/main_chat/response_continuation.py]: please pass db as class constructor parameter as it is done in many chatbot components which use database

File: src/main_chat/response_continuation.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    # ------------------------------------------------------
+    db.add_conversation(text="przykro mi, to wszystko co wiem", tag='cannot_say_more')
+    db.add_conversation(text="to wszystko co wiem na ten temat", tag='cannot_say_more')
+    db.add_conversation(text="nie wiem więcej", tag='cannot_say_more')
+    db.add_conversation(text="na tę chwilę to musi wystarczyć", tag='cannot_say_more')
+
+
+class ResponseContinuationHandler:
+    def __init__(self):
+        self.db = DatabaseProxy('mongodb://localhost:27017/', 'PepperChatDB')
```
Comment (rpych) [inline src/main_chat/response_continuation.py]: please pass db as class constructor parameter as it is done in many chatbot components which use database

**Pull request merged**